cdm/src/main/java/ucar/nc2/dataset/conv/CF1Convention.java

```diff
@@ -135,7 +135,19 @@ public void augmentDataset(NetcdfDataset ds, CancelTask cancelTask) throws IOExc
           v.addAttribute(new Attribute(_Coordinate.AxisType, AxisType.Ensemble.toString()));
           continue;
         }
-
+        
+        //sjk code
+        if (v.findAttribute(CF.GEOMETRY) != null) {
+        	v.addAttribute(new Attribute(CF.NODES, v.findAttIgnoreCase(CF.NODES, "nodes")));

```

Comment:
Looks good! Is it prohibitively time-consuming to add unit tests for this new functionality?